### DISCUSSION TITLE
https://maxkim-j.github.io/posts/olivia-rodrigo

### DISCUSSION THREAD

**Discussion by giscus** [MaxKim-J/maxkim-j.github.io#73] [Announcements]
# I'm not cool, I'm not smart, I can't even parallel park

올리비아 로드리고

http://localhost:8000/posts/olivia-rodrigo

**Comment by hanameee** [comment]
> 왜 이렇게 화내니...

ㅋㅋㅋㅋㅋㅋ데자뷰 가사 좀 묘하져... 로드리고는 연기도 해서 그런가 노래에서나 무대에서나 표현력이 좋아서 보는 재미가 이씀다. 인터뷰는 처음 봤는데 앞으로 만들 음악들도 기대되네염 내한해줘!!!!

**Reply by MaxKim-J** [reply]
>  연기도 해서 그런가 노래에서나 무대에서나 표현력이 좋아서 

오 맞네유 연기에서 온 것이겠어요 내한해줘!!!
